app/main.py

```diff
@@ -1,13 +1,31 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+def format_linter_error(
+        error: dict
+) -> dict:
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
-def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+def format_single_linter_file(
+        file_path: str,
+        errors: list
+) -> dict:
+    return {
+        "errors": [format_linter_error(error)
+                   for error in errors],
+        "path": file_path,
+        "status": "failed" if errors else "passed"
+    }
 
 
-def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+def format_linter_report(
```

Comment:
You need to wrap arguments on a new line if the length of the line with all the arguments does not satisfy the rules of flake8 or PEP8